agent/consul/usagemetrics/usagemetrics.go

```diff
@@ -36,6 +36,10 @@ var Gauges = []prometheus.GaugeDefinition{
 		Name: []string{"consul", "members", "servers"},
 		Help: "Measures the current number of server agents registered with Consul. It is only emitted by Consul servers. Added in v1.9.6.",
 	},
+	{
+		Name: []string{"consul", "kv", "entries"},
+		Help: "Measures the current number of server agents registered with Consul. It is only emitted by Consul servers. Added in v1.10.3.",
```

Comment:
Should be `Measures the current number of unique KV entries written in Consul. It is only emitted by Consul servers. Added in v1.10.3.`